modules/ducktests/tests/ignitetest/tests/self_test.py

```diff
@@ -53,3 +57,53 @@ def test_assertion_convertion(self, ignite_version):
         else:
             app.stop()
             assert False
+
+    def test_clock_sync(self):
+        """
+        Tests if clocks are synchronized between nodes.
+        """
+
+        service = GetTimeService(self.test_context, self.test_context.cluster.num_available_nodes())
```

Comment:
I think we should provide the check multiple times and compare max difference with `max_clock_spread_ms`.

Reply:
Done.